docker/docker-compose.yml

```diff
@@ -1,15 +1,27 @@
 services:
   postgres-old:
-    image: postgres:9.6  # Use the specific minor version closest to your dump
+    image: postgres:9.6
     environment:
       POSTGRES_DB: old_ckan_db
       POSTGRES_USER: postgres
       POSTGRES_PASSWORD: password
     ports:
-      - "9133:5432"  # Map host port 9133 to container port 5432 to avoid conflict with any local PG
+      - "9133:5432"
     volumes:
-      - postgres_data:/var/lib/postgresql/data
+      - postgres_old_data:/var/lib/postgresql/data
       - ./dump:/dump
 
+  postgres-new:
```

Comment:
The _new_ postgres instance will be a CKAN instance that is also running in the migrator user machine.
None of gthe changes of this compose file is required
We expect only to run a docker machine for the old DB. For the new one, we expect a CKAN instance running. We are not going to include here this, the user will use its own machine